challenge_two/src/utils/createContract.ts

```diff
@@ -0,0 +1,7 @@
+import { Contract, Provider } from "ethers";
+
+import { V3_FACTORY_ADDRESS, IUNISWAPV3FACTORY } from "./constants";
+
+export const getFactoryContract = (provider: Provider): Contract => {
+  return new Contract(V3_FACTORY_ADDRESS, IUNISWAPV3FACTORY, provider);
+};
```

Comment:
```suggestion
import { Contract, providers } from "ethers";

import { V3_FACTORY_ADDRESS, IUNISWAPV3FACTORY } from "./constants";

export const getFactoryContract = (provider: providers.Provider): Contract => {
  return new Contract(V3_FACTORY_ADDRESS, IUNISWAPV3FACTORY, provider);
};
```
ethers doesn't seem to have `Provider` imported